common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2752,6 +2752,11 @@
         "message" : [
           "A type of keys and values in `map()` must be string, but got <mapType>."
         ]
+      },
+      "NULL_VALUE" : {
+        "message" : [
+          "Values of keys and values in `map()` must be string, but got null in value."
```

Comment:
Users can build values dynamically, and could be not clear which one is `null`. Let's help them in troubleshooting, and provide `key` which has the `null` value.